control/autoware_trajectory_follower_node/src/controller_node.cpp

```diff
@@ -254,6 +286,75 @@ void Controller::publishProcessingTime(
   pub->publish(msg);
 }
 
+std::optional<ControlHorizon> Controller::createControlHorizon(
```

Comment:
The function name `createControlHorizon` does not clearly convey this functionality. It is better to include terms such as mergeLatLonHorizon or Resample to clarify its purpose.